script/proto_generate.sh

```diff
@@ -0,0 +1,19 @@
+git --version 2>&1 >/dev/null
+GIT_IS_AVAILABLE=$?
```

Comment:
You don't need to use the previous output here.  You can just do:
`GIT_IS_AVAILABLE=$(git --version 2>&1 >/dev/null)`